README.md

```diff
@@ -80,7 +145,7 @@ Calliope will serve the needs of most of our projects out of the box, but if you
 Calliope is just a fancy wrapper around Gulp, so all tasks that you may need to create will be just standard Gulp tasks. However, there are three different types of tasks within Calliope, some of which get special treatment.
 
 1. **Pipelines:** These are typically tasks that process static assets, e.g. compiling styles, compressing JavaScript, processing images, moving fonts around, etc. If your custom task or override is intended to be run as part of each build, it goes in `calliope/pipelines/`. In addition to creating custom pipelines, Calliope supports overriding its default pipelines if your project requires this. See [Overriding Default Pipelines] below for details.
-1. **Daemons:** Daemons are long-running processes that you want Calliope to run _alongside_ any watched tasks. Anytime you run `yarn calliope start`, Calliope will not only build assets and start watching for changes, but also run any daemons that may be available. If you need to run a service during development (a component library server, an API stubs server, etc.), it belongs in `calliope/daemons/`. While you may create custom daemons to run in parallel to build and watch tasks during development, the default reverse proxy daemon may not be overridden at this time.
+1. **Daemons:** Daemons are long-running processes that you want Calliope to run _alongside_ any watched tasks. Anytime you run `yarn calliope start`, Calliope will not only build assets and start watching for changes, but also run any daemons that may be available. If you need to run a service during development (a component library server, an API stubs server, etc.), it belongs in `calliope/daemons/`. While you may create custom daemons to run in parallel to build and watched tasks during development, the default reverse proxy daemon may not be overridden at this time.
```

Comment:
```suggestion
1. **Daemons:** Daemons are long-running processes that you want Calliope to run _alongside_ any watched tasks. Anytime you run `yarn calliope start`, Calliope will not only build assets and start watching for changes, but also run any daemons that may be available. If you need to run a service during development (a component library server, an API stubs server, etc.), it belongs in `calliope/daemons/`. While you may create custom daemons to run in parallel to build and watched tasks during development, the default reverse proxy daemon may not be overridden at this time.
```

Did you mean to make this change? I think it was right before. 🤔

Reply:
Yes, I’m making a point of saying “watched” tasks to refer to tasks whose files are watched and processed when changes are detected.

Reply:
Ah, got it. Maybe "quote" the words? Or maybe this doesn't confuse a front-ender? 🤷🏼‍♂️ 

Either way is fine with me. Feel free to resolve this.

Reply:
It’s not really front-end specific language. Just the way I chose to name tasks that are run anytime a watched file changes. I wanted to differentiate it from `watch` because previous versions of tooling have used a `watch` task (as in: `npm run watch` or `yarn watch`) and that is no longer a thing we do. We now do `start` to start watching _and doing other things_ like starting a reverse proxy and other daemons.

Thanks for making me think about it some more, though. Resolving this now.

Reply:
Glad I could help? possibly? 😂 

👍🏼